### DISCUSSION TITLE
Canva

### DISCUSSION THREAD

**Discussion by DanaMouyon99** [community/community#29655] [Other Feature Feedback, Questions, & Ideas]
Bonjour je ne trouve pas Canva pour lier mon compte à celui de Canva. pouvez-vous m'aider.

**Comment by melvincwng** [comment]
@DanaMouyon99, Salut, tu es étudiant ?

Si vous êtes étudiant, juste pour vous informer que Canva n'est plus disponible dans le GitHub Student Developer Pack.

J'espère que cela clarifie.

Translated:

Hi there, are you a student?

If you are a student, just to inform you that Canva is no longer available in the GitHub Student Developer Pack.

Hope this clarifies.
